redash/authentication/jwt_auth.py

```diff
@@ -21,7 +22,7 @@ def get_public_keys(url):
         if 'keys' in data:
             public_keys = []
             for key_dict in data['keys']:
-                public_key = jwt.algorithms.RSAAlgorithm.from_jwk(json.dumps(key_dict))
+                public_key = jwt.algorithms.RSAAlgorithm.from_jwk(json_dumps(key_dict))
```

Comment:
Not sure we should use `json_dumps` here.

Reply:
Why? `json_dumps` is just a pass through for with our JSONEncoder which won't get triggered with the payload here.

This is just a step to make sure we use `simplejson` everywhere and not the outdated Python 2.x stdlib library. Once we're on Python 3 we can revert the use of simplejson to the stdlib version of course.

Reply:
Just changing that to simplejson.dump instead.